physionet-django/console/templates/console/process_credential_application.html

```diff
@@ -78,25 +78,6 @@ <h2 style='font-size:20px;'>Reference comments:</h2>
         </div>
```

Comment:
Line 31 (`{% if application.credential_review.status == 50 %}`) should be updated so that the "Contact reference" button is displayed.